examples/zephyr/certificate_provisioning/src/main.c

```diff
@@ -140,13 +142,23 @@ int main(void)
     uint8_t *tls_client_key = NULL;
     size_t tls_client_crt_len, tls_client_key_len;
 
-    load_credential_from_fs(CLIENT_CERTIFICATE_PATH, &tls_client_crt, &tls_client_crt_len);
-    load_credential_from_fs(PRIVATE_KEY_PATH, &tls_client_key, &tls_client_key_len);
+    while (1)
+    {
+        int ret = load_credential_from_fs(CLIENT_CERTIFICATE_PATH, &tls_client_crt, &tls_client_crt_len);
+        if (ret > 0)
+        {
+            ret = load_credential_from_fs(PRIVATE_KEY_PATH, &tls_client_key, &tls_client_key_len);
+            if (ret > 0)
+            {
+                break;
+            }
+        }
```

Comment:
What if the first cert will be successfully loaded and the second one will fail? We'll be stuck in this loop forever, right?

Reply:
Yeah that'e the intention. We should hang in this loop until the use has loaded both certificates. But this comment reminded me I need to update the README, so I'll do that next!

Reply:
What about that case:
- `CLIENT_CERTIFICATE_PATH` is loaded successfully
- `PRIVATE_KEY_PATH` fails to load

in the next loop `CLIENT_CERTIFICATE_PATH` load will be attempted, but will most likely fail due to already being loaded. Or will it behave differently?

Reply:
That's a good callout! It will load fine, but we'll leak memory. I'll fix it 👍

Reply:
Fixed